packages/server-core/src/appconfig.ts

```diff
@@ -192,6 +193,7 @@ const authentication = {
       secret: process.env.FACEBOOK_CLIENT_SECRET!
     },
     github: {
+      appid: process.env.GITHUB_APP_ID!,
```

Comment:
This is added here but doesn't seem to be displayed in `XREngine/packages/client-core/src/admin/components/Setting/Authentication.tsx` file

Reply:
@zulqarnainhanif Yep. I just realized that we have an option to change the Authentication env on the admin page. Thank you so much!